packages/block-library/src/breadcrumbs/index.php

```diff
@@ -102,6 +116,48 @@ static function ( $item ) {
 	return $breadcrumb_html;
 }
 
+/**
+ * Adds pagination breadcrumb if on a paged view.
+ *
+ * Converts the last breadcrumb item to a link and adds "Page X" as the current page.
+ *
+ * @since 6.9.0
+ *
+ * @param array $breadcrumb_items Array of breadcrumb HTML items.
+ *
+ * @return array Modified breadcrumb items with pagination if applicable.
+ */
+function block_core_breadcrumbs_maybe_add_paged( $breadcrumb_items ) {
+	$paged = get_query_var( 'paged' );
```

Comment:
Nitpick: On packages/block-library/src/query-total/index.php we use with an explicit cast (int) get_query_var( 'paged', 1 ), maybe here we could use the same and then the if would be only $paged > 1